SitemapElementInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle;
+
+/**
+ * Documents persisted with phpcr-odm should implement that interface.
+ * They also need to create a property with the name "visible_for_sitemap" to be recognized
+ * by the default provider, that generates the UrlInformation object.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@gmx.de>
+ */
+interface SitemapElementInterface
```

Comment:
maybe we could drop this interface as we do not use it anyways, and could instead configure the phpcr-odm provider with a list of properties of which at least one must be true for a document to be included. if no properties are configured, there is no filtering and all routable documents end up in the sitemap.